<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>527</width>
    <height>183</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="editPartyNameLineEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="editPartyLeaderComboBox">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>90</y>
     <width>120</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="editPartyComboBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>90</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Seurue</string>
   </property>
  </widget>
  <widget class="QPushButton" name="editPartyCancelPushButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Peruuta</string>
   </property>
  </widget>
  <widget class="QPushButton" name="editPartySavePushButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>130</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Tallenna</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>100</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Seurueen nimi</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>70</y>
     <width>110</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Seurueenjohtaja</string>
   </property>
  </widget>
  <widget class="QPushButton" name="editPartyPopulatePushButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>30</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Täytä</string>
   </property>
  </widget>
  <widget class="QComboBox" name="editPartyTypeComboBox">
   <property name="geometry">
    <rect>
     <x>298</x>
     <y>90</y>
     <width>120</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>70</y>
     <width>110</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Seurueen tyyppi</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>editPartyComboBox</tabstop>
  <tabstop>editPartyPopulatePushButton</tabstop>
  <tabstop>editPartyNameLineEdit</tabstop>
  <tabstop>editPartyLeaderComboBox</tabstop>
  <tabstop>editPartyCancelPushButton</tabstop>
  <tabstop>editPartySavePushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
